src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.person;
+
+
+public class Contact {
+
+    public final String value;
+    private boolean isPrivate;
+
+    public Contact (String address, boolean isPrivate) {
+        String trimmedAddress = address.trim();
+        this.isPrivate = isPrivate;
+
+        this.value = trimmedAddress;
+    }
+
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.